rbac/management/group/view.py

```diff
@@ -340,3 +348,126 @@ def principals(self, request, uuid=None):
+    def roles(self, request, uuid=None):
+        """Get, add or remove roles from a group."""
+        """
+        @api {get} /api/v1/groups/:uuid/roles/   Get roles for a group
+        @apiName getRoles
+        @apiGroup Group
+        @apiVersion 1.0.0
+        @apiDescription Get roles for a group
+
+        @apiHeader {String} token User authorization token
+
+        @apiParam (Path) {String} id Group unique identifier.
+
+        @apiSuccess {String} uuid Group unique identifier
+        @apiSuccess {String} name Group name
+        @apiSuccess {Array} roles Array of roles
+        @apiSuccessExample {json} Success-Response:
+            HTTP/1.1 200 OK
+            {
+                "roles": [
+                    {
+                        "name": "RoleA",
+                        "uuid": "4df211e0-2d88-49a4-8802-728630224d15",
+                        "description": "RoleA Description"
+                    }
+                ]
+            }
+        """
+        """
+        @api {post} /api/v1/groups/:uuid/roles/   Add roles to a group
+        @apiName addRoles
+        @apiGroup Group
+        @apiVersion 1.0.0
+        @apiDescription Add roles to a group
+        @apiHeader {String} token User authorization token
+        @apiParam (Path) {String} id Group unique identifier
+        @apiParam (Request Body) {Array} roles Array of role UUIDs
+        @apiParamExample {json} Request Body:
+            {
+                "roles": [
+                    "4df211e0-2d88-49a4-8802-728630224d15"
+                ]
+            }
+        @apiSuccess {String} uuid Group unique identifier
+        @apiSuccess {String} name Group name
+        @apiSuccess {Array} roles Array of roles
+        @apiSuccessExample {json} Success-Response:
+            HTTP/1.1 200 OK
+            {
+                "roles": [
+                    {
+                        "name": "RoleA",
+                        "uuid": "4df211e0-2d88-49a4-8802-728630224d15",
+                        "description": "RoleA Description"
+                    }
+                ]
+            }
+        """
+        """
+        @api {delete} /api/v1/groups/:uuid/roles/   Remove roles from group
+        @apiName removeRoles
+        @apiGroup Group
+        @apiVersion 1.0.0
+        @apiDescription Remove roles from a group
+
+        @apiHeader {String} token User authorization token
+
+        @apiParam (Path) {String} id Group unique identifier
+
+        @apiParam (Query) {String} roles List of comma separated role UUIDs
+
+        @apiSuccessExample {json} Success-Response:
+            HTTP/1.1 204 NO CONTENT
+        """
+        roles = request.data.pop(ROLES_KEY, [])
+        group = self.get_object()
+        if request.method == 'POST':
+            self.add_roles(group, roles)
+            response_data = GroupRoleSerializer(group)
+        elif request.method == 'GET':
+            response_data = GroupRoleSerializer(group)
```

Comment:
Do you think you need to support any filtering of roles in the group? Or ordering of the roles in the group asc/desc? Don't need to add it with this PR, but worth thinking about and whether the UI will need that functionality.

Reply:
Yeah, I think that would be beneficial. I also logged something for pagination, which seems to be not quite as straightforward as other standard actions. I'm actually working through that now though, and was going to add a follow-up PR. I'll make note of filtering/ordering as well, thanks!